.docker/Dockerfile.nginx-drupal

```diff
@@ -5,7 +5,7 @@ FROM govcmsdev/nginx-drupal
 
 # nginx config.
 COPY .docker/images/nginx/helpers/ /etc/nginx/helpers/
-COPY .docker/images/nginx/x_frame_options.conf /etc/nginx/conf.d/drupal/location_drupal_append_x_frame_options.conf
+COPY .docker/images/nginx/x_frame_options.template /etc/nginx/conf.d/drupal/location_drupal_append_x_frame_options.template
```

Comment:
Nginx does not support env variables in config blocks. For this reason we move the config to a template, with env-variables to be replaced inline via `envsubst` during build

Reply:
@srowlands we currently just use `envplate` and any folders under `/etc/nginx/conf.d/` are automatically processed. Basically you can just put the env var directly into the conf file and they'll be replaced at runtime.

https://github.com/amazeeio/lagoon/blob/master/images/nginx/docker-entrypoint#L14

Reply:
Ah, that simplifies things a bit. I'll clean up shortly. 👍